### PR TITLE
Shapeit5 switch

### PR DESCRIPTION
**Pull request by LouisLeNezet**
## PR checklist
New module and last of the software SHAPEIT5.
The switch function allows to compare and detect the errors between the phasing estimate and the truth.
- [X] This comment contains a description of changes (with reason).
- [X] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [X] If necessary, include test data in your PR.
- [X] Remove all TODO statements.
- [X] Emit the `versions.yml` file.
- [X] Follow the naming conventions.
- [X] Follow the parameters requirements.
- [X] Follow the input/output options guidelines.
- [X] Add a resource `label`
- [X] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [X] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/shapeit5/switch/main.nf [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    label 'process_low'
+
+    conda "bioconda::shapeit5=1.0.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/shapeit5:1.0.0--h0c8ee15_0':
+        'quay.io/biocontainers/shapeit5:1.0.0--h0c8ee15_0'}"
+
+    input:
+        tuple val(meta), path(estimate), path(estimate_index), val(region), path(pedigree)
+        tuple val(meta2), path(truth), path(truth_index)
```
Comment (matthdsm) [inline modules/nf-core/shapeit5/switch/main.nf:12]: Can you align the comma's?

File: modules/nf-core/shapeit5/switch/main.nf [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    tuple val(meta), path("*.txt.gz")    , emit: errors
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args   = task.ext.args   ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def freq_cmd     = freq     ? "--frequency ${freq}"   : ""
```
Comment (matthdsm) [inline modules/nf-core/shapeit5/switch/main.nf:25]: alignment

**Review by matthdsm — APPROVED** [review]
LGTM, just need to fix the alignments

File: modules/nf-core/shapeit5/switch/main.nf (hunk 1 above)
Comment (LouisLeNezet) [reply modules/nf-core/shapeit5/switch/main.nf:12]: Sure !

**Review by matthdsm — APPROVED** [review]
🚀

**Pull request merged**